### PR TITLE
modify meta field, fix browserify

### LINKED ISSUE #77: Browserify returns empty module
When running `npm run build`, Browserify returns an empty module, instead of the library.

```
(function(f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g.urbitKeygen = f()}})(function(){var define,module,exports;return (function(){function r(e,n,t){function o(i,f){if(!n[i]){if(!e[i]){var c="function"==typeof require&&require;if(!f&&c)return c(i,!0);if(u)return u(i,!0);var a=new Error("Cannot find module '"+i+"'");throw a.code="MODULE_NOT_FOUND",a}var p=n[i]={exports:{}};e[i][0].call(p.exports,function(r){var n=e[i][1][r];return o(n||r)},p,p.exports,r,e,n,t)}return n[i].exports}for(var u="function"==typeof require&&require,i=0;i<t.length;i++)o(t[i]);return o}return r})()({1:[function(require,module,exports){

},{}]},{},[1])(1)
});


```

### PR DESCRIPTION
**Pull request by g-a-v-i-n**
- split `meta.generator` into two fields, `generatorName` and `generatorVersion`
- update tests
- Fix Browserify by using the old package.json syntax, resolves #77

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
   "main": "src/index.js",
-  "browser": "dist/index.js",
+  "browser": {
+    "src/index.js": "dist/index.js"
+  },
   "scripts": {
     "test": "nyc mocha --reporter spec test/test.js",
     "lint": "eslint src/",
-    "build": "mkdir -p dist && browserify src/index.js -s urbit-keygen > dist/index.js",
+    "build": "mkdir -p dist && browserify src/index.js -s urbit-keygen > dist/index.js --verbose",
```
Comment (Fang-) [inline package.json]: ?

**Comment by g-a-v-i-n** [comment]
The usecase is that I want to print the version of the generator on the paper wallets, but I don't want to have to do any string processing to the existing value `urbit-key-generation-v1.x.y` to separate the version number from the lib name.

**Comment by Fang-** [comment]
But the lib name is part of the version number? Like, it matters if it's urbit-key-generation version 1, or urbit-key-generation-hoon version 1.

(I don't actually feel super strongly though, I won't stop you.)

**Comment by g-a-v-i-n** [comment]
Sounds great

**Comment by g-a-v-i-n** [comment]
muchos gracias

**Comment by jtobin** [comment]
On Thu, Aug 29, 2019 at 12:03:17PM -0700, g-a-v-i-n wrote:
> muchos gracias

de nada, señor

**Pull request merged**